src/js/me-i18n.js

```diff
@@ -1,169 +1,226 @@
+					str
+				;
+
+
+				// Fetch the localized version of the string
+				if (i18n.locale.strings && i18n.locale.strings[language]) {
+					str = i18n.locale.strings[language][input];
+				}
+
+				// Fallback to default language if requested uid is not translated
+				if (!str && i18n.locale.strings && i18n.locale.strings[i18n['default']]) {
+					str = i18n.locale.strings[i18n['default']][input];
+				}
+
+				// As a last resort, use the requested uid, to mimic original behavior of i18n utils (in which uid was the english text)
+				str = str || input;
+
+				return str;
+
+			}
+
+			return input;
+		}
+
+	};
+
+	// i18n fixes for compatibility with WordPress
+	// if (typeof mejsL10n !== 'undefined') {
+	// 	i18n.locale.language = mejsL10n.language;
+	// }
+
+	// Register variable
+	mejs.i18n = i18n;
+
+	/**
+	 * Convert string using an algorithm to detect callbacks that will modify a string based on filters.
+	 *
+	 * This method will change a string with format:
+	 *  - '{0} {0|plural:second:seconds} left to finish playing and {1} {1|plural:second:seconds} to go'.format(1, 10)
+	 * to:
+	 * - '1 second left to finish playing and 10 seconds to go'
+	 *
+	 * where:
+	 * - {[0,1,2...]}  is the number to be placed from the method's arguments in respective order
+	 * - {[0,1,2...]|[filterName]:[replacement1]:[replacement2]:...} is the string to be placed based on the specified filter
+	 *
+	 * @see http://stackoverflow.com/questions/1353408/messageformat-in-javascript-parameters-in-localized-ui-strings
+	 * @return {String}
+	 */
+	String.prototype.format = function () {
+		var args = arguments;
+
+		return this.replace(/\{((\d+)((\|\w+(:\w+)*)*))\}/g, function() {
+			var arg = args[arguments[2]],
+				filters = arguments[3].split('|'),
+				i,
+				total,
+				curFilter,
+				curFilterArgs,
+				curFilterFunc,
+				defaultFilters = {}
+			;
+
+			// Find current language's rules to filter; otherwise, use default
+			for (i = 0, total = mejs.i18n.rules.length; i < total; i++) {
+				var rule = mejs.i18n.rules[i];
+				if (rule.languages.indexOf(mejs.i18n.getLanguage()) > -1) {
+					for (var property in rule) {
+						if (rule.hasOwnProperty(property) && property !== 'languages') {
+							defaultFilters[property] = rule[property];
+						}
+					}
+					break;
+				}
+			}
+
+			for(i = 0, total = filters.length; i < total; ++i) {
+				curFilterArgs = filters[i].split(':');
+				curFilter = curFilterArgs.shift();
+				curFilterFunc = defaultFilters[curFilter];
+
+				if(typeof curFilterFunc === 'function') {
+					arg = curFilterFunc.apply(null, [ arg ].concat(curFilterArgs));
+				}
+			}
+			return arg;
+		});
+	}
+
+}(document, window, mejs));
 
 // i18n fixes for compatibility with WordPress
-;(function(exports, undefined) {
-
-    "use strict";
-
-    if ( typeof mejsL10n != 'undefined' ) {
-        exports[mejsL10n.language] = mejsL10n.strings;
-    }
-
-}(mejs.i18n.locale.strings));
+// ;(function (mejs, undefined) {
```

Comment:
Maybe remove this whole section instead of commenting, then informat the Wordpress team of the modification ?